src/nft/components/explore/CarouselCard.tsx

```diff
@@ -295,13 +306,16 @@ const CollectionName = styled(ThemedText.MediumHeader)`
 `
 
 const CarouselCardHeader = ({ collection }: { collection: TrendingCollection }) => {
-  const theme = useTheme()
+  const [averageBanerImageColor, setAverageBanerImageColor] = useState('#fff')
+  useEffect(() => {
+    getColorFromUriPath(collection.bannerImageUrl).then((c) => c && setAverageBanerImageColor(c))
+  }, [collection.bannerImageUrl])
   return (
     <CardHeaderContainer src={collection.bannerImageUrl}>
       <CardHeaderColumn>
         <CollectionImage src={collection.imageUrl} />
         <CollectionNameContainer>
-          <CollectionName color={theme.accentTextLightPrimary} fontWeight="500">
+          <CollectionName style={{ color: readableColor(averageBanerImageColor) }} fontWeight="500">
```

Comment:
could you add this style to the styled component `CollectionName` and pass in the color?

Reply:
this also not effective for BAYC or MAYC, the text is dark on a dark background

Reply:
yeah, this isn't a perfect solution for i.e., collections w/ a light enough average color that `readableColor` selects black, even though the actual text render area on the banner image is v dark :/

Reply:
+1 on hiding this behind a styled components interface! Also, consider prefixing a custom property with a `$` so that it doesn't propagate to children (generally good practice for props that are Styled Components exclusive and used for styling)

Reply:
Honestly I think the Cal's solution of making the color explicitly white for both modes would look better for the majority of cases. https://uniswapteam.slack.com/archives/C04BVC4J5HU/p1669653436830549